### PR TITLE
livestock screening

### PR DESCRIPTION
**Pull request by AKostiv8**

### CODE REVIEW

File: scripts/ento/ento_reports/app.R [hunk 1]
```diff
@@ -109,19 +165,19 @@ server <- function(input, output, session) {
...
+  #   if (is_page("/")) {
+  #     runjs('$( "#household_pageId" ).addClass( "active" );')
+  #     runjs('$( "#livestock_pageId" ).removeClass( "active" );')
+  #     
+  #     # change_page("side")
+  #   } else if (is_page("livestock")) {
+  #     runjs('$( "#livestock_pageId" ).addClass( "active" );')
+  #     runjs('$( "#household_pageId" ).removeClass( "active" );')
+  #   }
+  # })
```
Comment (katekimani) [inline scripts/ento/ento_reports/app.R:180]: Please clean up the commented out code.

**Review by katekimani — CHANGES_REQUESTED** [review]
Looks good for the livestock screening. Was expecting the data collection tool parts as well though.

Format changes requested inline

File: scripts/ento/ento_reports/app.R (hunk 1 above)
Comment (AKostiv8) [reply scripts/ento/ento_reports/app.R]: @katekimani Data collection in progress. Requested changes are done.

**Comment by AKostiv8** [comment]
@katekimani Check again, please. I have integrated the questionnaires part to the app

**Review by katekimani — CHANGES_REQUESTED** [review]
Few calculations and alerts missing in the livestock

File: scripts/ento/ento_reports/pages/mod_questionnaires.R [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+            # mutate(percent_of_target_165 = NA) %>%
+            mutate(final_comp = if_else((le_1st_visit_gr + le_no_first_visit_gr + le_followup_gr + le_no_followup_gr) == le_visited_gr, 1, 0)) 
+          if(nrow(df) > 0) {
+            df <- df %>%
+              add_row(cluster = "Total", 
+                      le_visited_gr = sum(.$le_visited_gr, na.rm = TRUE),
+                      le_1st_visit_gr = sum(.$le_1st_visit_gr, na.rm = TRUE),
+                      le_no_first_visit_gr = sum(.$le_no_first_visit_gr, na.rm = TRUE),
+                      le_followup_gr = sum(.$le_followup_gr, na.rm = TRUE),
+                      le_no_followup_gr = sum(.$le_no_followup_gr, na.rm = TRUE),
```
[the remark this answers is not in the document]
Comment (AKostiv8) [reply scripts/ento/ento_reports/pages/mod_questionnaires.R:368]: @katekimani I know, that the alert is missed - I separated it into another task yesterday ( https://trello.com/c/KXeXdQsG/345-connect-alert-to-questionnaires-part-communication-between-modules ) It is better to make after all modules are created as it requires "communication" between. modules

**Pull request merged**